### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.NullPointerException

### ISSUE THREAD

**Issue by zoemaas** [microsoft/azure-tools-for-java#6687]
IntelliJ build version: 2022.1.2 IU-221.5787.30
OS: Linux
JDK: JetBrains s.r.o. 11.0.15
Plugin version: 3.65.1-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,37,1247x759,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.NullPointerException
	at com.intellij.javascript.debugger.sourcemap.visualizer.SourcemapFileViewer.<init>(SourcemapFileViewer.kt:47)
	at com.microsoft.azure.toolkit.intellij.appservice.actions.AppServiceFileAction.openFileInEditor(AppServiceFileAction.java:125)
	at com.microsoft.azure.toolkit.intellij.appservice.actions.AppServiceFileAction.lambda$openAppServiceFile$4(AppServiceFileAction.java:104)
	at com.microsoft.azure.toolkit.lib.common.task.AzureTask.lambda$new$0(AzureTask.java:117)
	at com.microsoft.azure.toolkit.lib.common.operation.AzureOperationAspect.execute(AzureOperationAspect.java:95)
	at com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.lambda$runInObservable$0(AzureTaskManager.java:384)
	at com.microsoft.azure.toolkit.lib.common.operation.OperationThreadContext.run(OperationThreadContext.java:81)
	at com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager.lambda$runInObservable$1(A

**Comment by wangmingliang-ms** [comment]
This issue is raised from the JavaScript plugin. were you trying to open a JavaScript file having source map? 
We cannot reproduce this issue when opening a JS source map file in IDE and We haven't observed this exception in the last 2 months. 
So, I believe it has been fixed by IntelliJ. 
Can you try the latest versions of IDE and our plugin?

**Comment by wangmingliang-ms** [comment]
I can reproduce it now, it should be fixed by commit eb033d6f, and this fix will be included in the next stable release, thanks for you help.

**Comment by zoemaas** [comment]
Sorry for the late reply. I'm using the latest versions and the issue didn't show up lately. I'm happy to help. Thanks!

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
